src/www/ui/page/AdminGroupUsers.php

```diff
@@ -152,7 +152,7 @@ function setNewPermUrl(newperm){
     return $this->render('admin_group_users.html.twig', $this->mergeWithDefault($vars));
   }
 
-  private function updateGUMPermission($gum_pk, $perm)
+  public function updateGUMPermission($gum_pk, $perm)
   {
     $dbManager = $this->getObject('db.manager');
```

Comment:
Please get DBManager from the parameters. The containers for UI and REST API are different and can cause issues.
```suggestion
  public function updateGUMPermission($gum_pk, $perm, $dbManager)
  {
```

Reply:
@dushimsam can you please do the required changes?

Reply:
sorry sir I don't really know how i missed the change 😂
thanks for reminding me.

Reply:
hope now everything is fine.

Reply:
@dushimsam , now you need to pass the `dbmanager` where the function is called. Like at following location:
https://github.com/fossology/fossology/blob/69ea7f35164245fe4ede460b0caa4864425975b7/src/www/ui/page/AdminGroupUsers.php#L69

Reply:
Reminder @dushimsam :-)

Reply:
I think it's there sir , 
see 
![image](https://user-images.githubusercontent.com/66276301/188405470-70e37f6b-f82d-4223-ab3f-fef3a15d2c91.png)

Reply:
Still missing in the commit, please have a look:
https://github.com/fossology/fossology/blob/0bebbc8a462edc27b3f90ddf1f489b96617167cb/src/www/ui/page/AdminGroupUsers.php#L69

Reply:
ahhh sorry I just forgot to update where it was being called previously

Reply:
Resolved, you can check now.